generator/generator.md

```diff
@@ -0,0 +1,246 @@
+| comments            | Additional comments for the question (optional)         | If number of persons = 7+                                                     | string?              |
+
+> <span id="input">**Note:**</span> The `inputType` field specifies the type of input for the question and can be one of the following: Checkbox, Custom, NextButton, Number, Radio, Range, String, Text, or TextArea.
+
+> <span id="cond">**Note:**</span> The `conditional` field allows you to define conditional logic for displaying the widget based on other responses. For example, you can specify a condition like `nbPersonnesSeptPlusConditional` to show the widget only if the number of people is 7 or more.
+
+> <span id="val">**Note:**</span> The `validation` field is optional and allows you to define validation logic for the widget. In the example, `moreOrEqualTo7Validation` signifies that the widget will be considered valid if the entered value is equal to or greater than 7.
+
+> <span id="choices">**Note:**</span> The `choices` field is optional, but relevant for `Radio` and `Checkbox` inputs and allows you to specify the available choices for the question.
+
+> <span id="range">**Note:**</span> The `inputRange` field is optional and allows you to define the valid range of values for `Range` input.
+
+### Widgets TypeScript
+
+```typescript
+// home/widgets.tsx
+export const home_Q2_nbPersonnesResidenceTravailPreciser: inputTypes.InputString = {
+    ...defaultInputBase.inputNumberBase,
+    path: 'home.Q2_nbPersonnesResidenceTravailPreciser',
+    label: (t: TFunction) => t('home:home.Q2_nbPersonnesResidenceTravailPreciser'),
+    conditional: customConditionals.nbPersonnesSeptPlusConditional,
+    validations: validations.moreOrEqualTo7Validation
+};
+```
+
+### Widgets Example
+![Widgets example](./assets/images/widgetsExample.png)
+
+## Generate Conditionals
+
+In your survey logic, conditionals play a key role in determining if the widget will appear or not. The table below outlines the fields used to define conditionals in Conditionals tab, along with an example and the corresponding TypeScript code.
+
+### Conditionals Fields
+
+| Field                       | Description                            | Example                       | Type                    |
+|-----------------------------|----------------------------------------|-------------------------------|-------------------------|
+| conditionalName             | Name of the conditional                | siPermisConduireConditional   | string                  |
+| logicalOperator             | Logical operator (optional)            | &&                            | && or \|\|              |
+| path                        | Path to the data point for comparison  | [${relativePath}](#rel).age   | string                  |
```

Comment:
I don't understand the example here. Also "in the responses object" instead of "data point"